components/admin_console/team_channel_settings/channel/list/channel_list.tsx

```diff
@@ -31,7 +32,7 @@ interface ChannelListProps {
     data: ChannelWithTeamData[];
     total: number;
     removeGroup?: () => void;
-    onPageChangedCallback?: () => void;
+    onPageChangedCallback?: (state: ChannelSettingsState) => void;
```

Comment:
This prop doesn't appear to actually be used, so perhaps we could remove it? But if we remove that, we're removing all the state from `ChannelSettings` so I'm not sure if I'm missing something here.

Reply:
It seems to be ok removing onPageChangedCallback from both ChannelList and ChannelSettings.